EssentialFeed/EssentialFeed/Image Comment/Image Comment Presentation/ImageCommentsPresenter.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ImageCommentsPresenter.swift
+//  EssentialFeed
+//
+//  Created by Alok Subedi on 03/02/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public protocol ImageCommentsLoadingView {
+	func display(_ viewModel: ImageCommentsLoadingViewModel)
+}
+
+public protocol ImageCommentsErrorView {
+	func display(_ viewModel: ImageCommentsErrorViewModel)
+}
+
+public protocol ImageCommentsView {
+	func display(_ viewModel: ImageCommentsViewModel)
+}
+
+public class ImageCommentsPresenter {
+	private let loadingView: ImageCommentsLoadingView
+	private let errorView: ImageCommentsErrorView
+	private let imageCommentsView: ImageCommentsView
+	
+	public init(imageCommentsView: ImageCommentsView, loadingView: ImageCommentsLoadingView, errorView: ImageCommentsErrorView) {
+		self.imageCommentsView = imageCommentsView
+		self.loadingView = loadingView
+		self.errorView = errorView
+	}
+	
+	public static var Title: String {
```

Comment:
The convention is to start property names with a lowercase letter:

```suggestion
	public static var title: String {
```